src/webgpu/api/operation/memory_sync/buffer/buffer_sync_test.ts

```diff
@@ -0,0 +1,253 @@
+import { assert } from '../../../../../common/framework/util/util.js';
+import { GPUTest } from '../../../../gpu_test.js';
+
+const kSize = 4;
+
+export class BufferSyncTest extends GPUTest {
```

Comment:
@kainino0x , you mentioned that it is not necessary to put helper functions into a fixture. Do you mean that we don't need to define `class BufferSyncTest`? In that case, how can I include the helper functions into ww.spec.ts?

Reply:
Done.